src/zhinst/toolkit/driver/modules/shfqa_sweeper.py

```diff
@@ -70,8 +70,12 @@ def __init__(self, daq_server: ziDAQServer, session: "Session"):
             "force_sw_trigger": "sw_trigger_mode",
         }
         super().__init__(self._create_nodetree(), tuple())
-        self._daq_server = daq_server
-        self._raw_module = CoreSweeper(daq_server, "")
+        self._daq_server = ziDAQServer(
+            session._daq_server.host,
+            session._daq_server.port,
+            6,
```

Comment:
```suggestion
            session.server_host,
            session.server_port,
            6,
```
**Non-Blocking** (at least use the public function)